.github/integration_test.sh

```diff
@@ -17,7 +17,7 @@ if ! lein with-profile -user,-dev,+ci install; then
   exit 1
 fi
 
-if ! clojure -Ttools install nvd-clojure/nvd-clojure '{:mvn/version "RELEASE"}' :as nvd; then
+if ! clojure -Ttools install io.github.rm-hull/nvd-clojure '{:local/root "."}' :as nvd; then
```

Comment:
What does `io.github.rm-hull` exactly do/add? 

I tend to run `.github/integration_test.sh` locally often. Having `io.github.rm-hull` there invites one to think one might be running something else (as much as I can read `root .`)